src/knative/service.ts

```diff
@@ -22,10 +22,14 @@ import { executeInTerminal } from '../kn/knExecute';
 // }
 
 export default class Service extends KnativeItem {
```

Comment:
`knativeItem` didn't do anything before and does nothing now that you emptied it, so why keep the extend?